src/credit-card-number-field.css.ts

```diff
@@ -5,12 +5,31 @@ export const flip = style({
     perspective: 1000,
 });
 
-export const flipFront = sprinkles({position: 'absolute'});
+const baseIconContainer = style([
+    sprinkles({
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        height: '100%',
+    }),
+    {
+        backfaceVisibility: 'hidden',
```

Comment:
Not adding `backfaceVisibility: 'hidden'` to flipFront was causing weird behaviour in some devices